Test/multilabel_test.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""
+>>> from pycm import *
+>>> mlcm = MultiLabelCM([{'dog'}, {'cat', 'dog'}], [{'cat'}, {'cat'}])
+>>> len(mlcm)
+2
+>>> print(mlcm)
+pycm.MultiLabelCM(classes: ['cat', 'dog'])
+>>> mlcm
+pycm.MultiLabelCM(classes: ['cat', 'dog'])
+>>> mlcm.samplewise_cm(0)
+pycm.ConfusionMatrix(classes: [0, 1])
+>>> mlcm.samplewise_cm(0)
+pycm.ConfusionMatrix(classes: [0, 1])
```

Comment:
Is this duplication necessary, @sepandhaghighi ?

Reply:
Yeah, to check the memory-based return.